src/DayColumn.js

```diff
@@ -140,12 +140,14 @@ let DaySlot = React.createClass({
       , eventWrapperComponent: EventWrapper
       , rtl: isRtl
       , step
+      , timeslots
```

Comment:
This will be `undefined ` (and mess up the styling) unless it's explicitly added as a prop to the Calendar. The default value of 2 seems to be set as default only in TimeColumn and TimeSlotGroup.